doc/schema-hoomd.rst

```diff
@@ -12,12 +12,12 @@ of the existing data chunks. Any newer reader will initialize new data chunks wi
 not present in an older version file.
 
 :Schema name: ``hoomd``
-:Schema version: 1.3
+:Schema version: 1.4
 
 Use-cases
 ---------
 
-There are a few problems with XML, DCD, and other dump files that the GSD schema ``hoomd`` solves.
+The GSD schema ``hoomd`` provides.
```

Comment:
```suggestion
The GSD schema ``hoomd`` provides:
```

Reply:
The contents of the list below don't finish the sentence started above with "provides."